modules/ROOT/pages/db/db-connector-insert-update-delete.adoc

```diff
@@ -3,48 +3,157 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The connector also provides insert, update and delete operations. Just like the xref:db/db-connector-query.adoc[select operation], these operations support dynamism and parameterization by using embedded DataWeave transformations:
+The Database connector supports insert, update, and delete operations.
+Similar to the SELECT operation, these operations support dynamic queries and
+parameterization by using embedded DataWeave transformations while also
+supporting `fetchSize`, `maxRows`, and timeout parameters. Although examples of input
+parameterization, dynamic queries, and parameter usage are provided, you can consult
+the xref:db/db-connector-query.adoc[SELECT operation documentation]
+for more information.
+
+== Insert Example
+
+Suppose there is a database schema named *Products* that has a table named
+*electronic*. *electronic* contains columns named *id*, *name*, *description*, *price*, and *discount*.
+The following SQL statement creates the table:
+
+[source,sql,linenums]
+----
+CREATE TABLE electronic(
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(100),
+    description VARCHAR(255),
+    price SMALLINT,
+    discount TINYINT
+)
+----
+
+For an example on creating a table using the Database connector, check how to
+xref:db/db-connector-ddl.adoc[Execute DDL Statements].
+
+Given that the table is already created and that a there is a
+xref:db/db-configure-connection.adoc[connection configuration] named `dbConfig`,
+the following operation inserts a new record in the *electronic* table:
 
 [source,xml,linenums]
 ----
 <db:insert config-ref="dbConfig">
   <db:sql>
-    INSERT INTO PLANET(POSITION, NAME, DESCRIPTION) VALUES (777, 'Pluto', :description)
+    INSERT INTO electronic(name, description, price, discount)
+    VALUES ('Coffee Machine', 'Model: XYZ99. Uses small size capsules.', 120, 5)
   </db:sql>
-  <db:input-parameters>
-    #[
-    {'description' : payload}
-    ]
-  </db:input-parameters>
 </db:insert>
-view rawinsert.xml hosted with ❤ by GitHub
+----
+
+Note that the *id* parameter is not passed in the query because it is an
+autoincrement key that is generated automatically by the database.
+
+[NOTE]
+=============
+To benefit from the advantages of
+xref:db/db-connector-query.adoc#_use_input_parameters_to_protect_database_queries[input parameters],
+the equivalent query should be written as follows:
+[source,xml,linenums]
+----
+<db:insert config-ref="dbConfig">
+    <db:sql>
+        INSERT INTO electronic(name, description, price, discount)
+        VALUES (:name, :description, :price, :discount)
+    </db:sql>
+    <db:input-parameters>#[{
+        name: 'Coffee Machine',
+        description: 'Model:XYZ99. Uses small size capsules.',
+        price: 120,
+        discount: 5}]
+    </db:input-parameters>
+</db:insert>
+----
+=============
+
+The `insert` operation will output a `Statement Result` object that contains 2 fields: a _affectedRows_
```

Comment:
Note a third presentation style here for operation names; standardize as much as possible.

will output > outputs

2 fields > two fields

a _affectedRows_ > an _affectedRows_

Reply:
Removed `` and implemented other feedback.